bindgen/ir/comp.rs

```diff
@@ -663,12 +680,23 @@ where
         unfilled_bits_in_unit = data_size - unit_size_in_bits;
     }
 
+    let final_bit_size = if let Some(next_field) = next_non_bitfield {
+        next_field
+            .offset()
+            .map_or(unit_size_in_bits, |off| off - bitfields_bit_begin_off)
+    } else if let Some(parent_layout_) = parent_layout {
```

Comment:
Trying to remove `parent_layout` doesn't cause any layout test failures at a glance. So I think it's fine to remove.